crates/but-core/src/unified_diff.rs

```diff
@@ -215,6 +221,21 @@ impl UnifiedDiff {
     }
 }
 
+fn compute_line_changes(hunks: &Vec<DiffHunk>) -> (u32, u32) {
```

Comment:
@Byron is there a better way / place to do this?

Reply:
I think this is a fine way - straightforward and it actually re-uses work that was already done (i.e. the patch itself).
I don't think I would have done it differently.